docs/editors-guide/generate-report.md

```diff
@@ -40,6 +40,10 @@ Which will generate a TSV report here: `reports/report-obsoletion-candidates.tsv
 
 The only requirement for a report is, that a suitable SPARQL query can be found in `../sparql/reports/`, for example, `../sparql/reports/obsoletion-candidates.sparql`. 
 
+Note:
+If you want to run a query using the "inferred" version of the ontology (e.g. harrisonview-termlist requires to check whether any parents (asserted and inferred) have the 'harrisonview' tag), use the following command: 
+sh run.sh make report-reason-query-
```

Comment:
if you add these cute little things around your text, it will show up with cool formatting:
it is the button next to the 1: 
`sh run.sh make report-reason-query-`

Does that make sense? if you try to edit this comment, you can see how it is formatted.

Can you use these when writing text that should be a command? Thanks!